setup.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+from os import path
+from setuptools import setup, find_packages
+
+
+pkg_name = "intanrhsreader"
+here = path.abspath(path.dirname(__file__))
+print(here)
+
+with open(path.join(here, "README.md"), "r") as f:
+    long_description = f.read()
+
+with open(path.join(here, "requirements.txt")) as f:
+    requirements = f.read().splitlines()
+
+with open(path.join(here, "version.py")) as f:
+    exec(f.read())
+
+setup(
+    name=pkg_name.replace("_", "-"),
+    version=__version__,
+    description="Intan rhs single data reader",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author="DataJoint",
+    author_email="info@datajoint.com",
+    license="MIT",
```

Comment:
Do we need to use the Intan GPL 3 license?

Reply:
Absolutely.

Reply:
Made the change.